src/imitation/util/reward_wrapper.py

```diff
@@ -1,24 +1,32 @@
+"""Common wrapper for adding custom reward values to an environment."""
+import numpy as np
 from stable_baselines.common.vec_env import VecEnvWrapper
 
 
-# TODO(shwang, qxcv): Keep Sam's version when merging #54.
 class RewardVecEnvWrapper(VecEnvWrapper):
-
-  def __init__(self, venv, reward_fn):
+  def __init__(self, venv, reward_fn, *, include_steps=False):
     """A RewardVecEnvWrapper uses a provided reward_fn to replace
     the reward function returned by `step()`.
 
-    Automatically resets the inner VecEnv upon initialization.
-    A tricky part about this class keeping track of the most recent
-    observation from each environment.
+    Automatically resets the inner VecEnv upon initialization. A tricky part
+    about this class is keeping track of the most recent observation from each
+    environment.
+
+    Will also include the previous reward given by the inner VecEnv in the
+    returned info dict under the `wrapped_env_rew` key.
 
     Args:
         venv (VecEnv): The VecEnv to wrap.
         reward_fn (Callable): A function that wraps takes in arguments
             (old_obs, act, new_obs) and returns a vector of rewards.
+        include_steps (bool): should reward_fn be passed an extra keyword
```

Comment:
I'd lean towards always passing this argument, and just having some reward functions ignore it (you can always wrap a function to drop that argument). Don't have a strong preference on this, though.

Reply:
The optional extra kwarg smells funny to me too. I'll add this to #31, but I think i'll leave this for now because this API change isn't related the current PR.